api/types/stores.go

```diff
@@ -16,3 +16,7 @@ type SockStore interface {
 	DeleteSock(sockID int) error
 	CountSocks() (int, error)
 }
+
+type OrderStore interface {
+	GetOrdersByStatus(status string) ([]Order, error)
```

Comment:
You'll also need a `GetOrders() ([]Order, error)` to handle cases without status constraint. OR you can just check `if status == ""` and run a different query without the `WHERE status = ...` clause.